frontend/src/views/WelcomePage.tsx

```diff
@@ -0,0 +1,42 @@
+import useAuth from "@hooks/useAuth";
+import { useNavigate } from "react-router-dom";
+
+/**
+ * Component representing the welcome page after user login.
+ *
+ * This component displays a welcome message, a logout button, and a button
+ * to navigate to the Dog List page.
+ *
+ * @component
+ * @returns The WelcomePage component.
+ */
+const WelcomePage = () => {
+  const { forceLogout } = useAuth();
+  const navigate = useNavigate();
+
+    /**
+   * Function to handle the "Dog List" button click.
+   *
+   * This function uses the React Router `useNavigate` hook to navigate to
+   * the Dog List page when the button is clicked.
+   *
+   * @function
+   */
+  const handleListButton = () => {
+    navigate("/list");
```

Comment:
I think you should create some list of paths. For this example navigate should look something like that:
```
navigate(paths.list)
```